index.html

```diff
@@ -3,102 +3,103 @@
 </head>
 
 <body>
 
-<header class="z">
-  <nav class="top-nav">
-    <div class="z__container">
-      <div class="foo-flex">
-        <h1 class="foo-bar"><a href="#">Some Student</a></h1>
+<header class="hero">
+  <nav class="hero__fill">
+    <div class="hero__container">
+      <div class="nav-flex">
+        <h1 class="nav-group"><a href="#">Some Student</a></h1>
 
-        <ul class="foo-bar"
-          <li class="foo-bar__x"><a href="#">Home</a></li>
-          <li class="foo-bar__x"><a href="#">About</a></li>
-          <li class="foo-bar__x"><a href="#">Contact</a></li>
+        <ul class="nav-group">
+          <li class="nav-group__item"><a href="#">Home</a></li>
+          <li class="nav-group__item"><a href="#">About</a></li>
+          <li class="nav-group__item"><a href="#">Contact</a></li>
         </ul>
       </div>
     </div>
   </nav>
 
-  <div class="z__fill flex-v-center">
-    <div class="z__container">
+  <div class="hero__fill nav-flex">
+    <div class="hero__container">
       <h2>
         This is my portfolio!
-      h2>
+      </h2>
       <h2>
         There may be many like it, so I should replace this text to show that I am reading.
       </h2>
       <h2>
-        <a href="mailto:my-email@goes-here.com" class="is-this-a-link">Contact Me?
+        <a href="mailto:my-email@goes-here.com" class="btn">Contact Me?</a>
       </h2>
     </div>
   </div>
 </header>
 
-<section class="y y--a">
-  <div class="y__container">
-    <h2 class="title title--jumbo">I'm a student at the Iron Yard</h2>
+<section class="section">
+  <div class="section--info">
+    <h2 class="section__header">I'm a student at the Iron Yard</h2>
 
-    <p class="title title--normal">I should write something about myself here and make it crafty.</p>
+    <p class="fancy-text">I should write something about myself here and make it crafty.</p>
   </div>
 </section>
 
-<section class="y>
-  <div class="y__container">
-    <h3 class="y__header">My First Day</h3>
+<section class="section">
+  <div class="section__container">
+    <h3 class="section__header">My First Day</h3>
 
     <hr class="line">
 
     <p>Day one is hard. It should be tough and challenging. I'll write something about it here. Two paragraphs would be nice.</p>
   </div>
 </section>
 
-<section class="y y--b">
-  <div class="y__container">
-    <h2 class="title title--jumbo">This is my tagline</h2>
+<section class="section--photo">
+  <div class="section__container">
+    <h2 class="fancy-text--big">This is my tagline</h2>
   </div>
 </section>
 
-<section classy y--c">
-  <div class="y__container">
-    <div class="notsure">
-      <div class="notsure__yeah notsure__yeah-?-?">
-        <div class="c">
-          <figure class="instagram">
+<section class="section--more">
+  <div class="section__container">
+    <div class="divider">
+      <div class="polaroid">
+        <div class="polaroid--before">
+          <figure class="polaroid--after">
```

Comment:
Check these class names for `polaroid`, we'll cover it later this week, but `::before` and `::after` are valid CSS selectors, but they are special "pseudo-elements" so you won't have `before` or `after` in your actual HTML.